src/buildCommon.js

```diff
@@ -80,9 +80,8 @@ const makeSymbol = function(
             metrics.width, classes);
     } else {
         // TODO(emily): Figure out a good way to only print this in development
-        typeof console !== "undefined" && console.warn(
-            "No character metrics for '" + value + "' in style '" +
-                fontName + "'");
+        typeof console !== "undefined" && console.warn("No character metrics " +
+            `for '${value}' in style '${fontName}' and mode '${mode}'`);
```

Comment:
I wish that our tests actually tested which error messages are being generated for the `.not.toParse()` and `.not.toBuild()` test cases.  Not something we should worry about in this PR.  I'll open an issue for this.